gensrc/thrift/PaloBrokerService.thrift

```diff
@@ -126,10 +127,17 @@ struct TBrokerPReadRequest {
     4: required i64 length;
 }
 
+enum EWhence {
```

Comment:
why EWhence, change to TSeekWhence

T means thrift

Reply:
i had changed